Updating release notes (#2712)
ROCm/ROCm: RELEASE.md

#### End of support of specific distros

The following distros are planned for end of support in ROCm 6.2:

- Ubuntu 20.04.5
- SLES 15 SP4
- RHEL 7.9
- CentOS 7.9

#### Documentation

CMake support is added for the documentation in the
[ROCm repository](https://github.com/RadeonOpenCompute/ROCm).

#### AMD Instinct™ MI50 end-of-support notice

AMD Instinct MI50, Radeon Pro VII, and Radeon VII products (collectively gfx906 GPUs) enter
maintenance mode in ROCM 6.0.0.

As outlined in [5.6.0](https://rocm.docs.amd.com/en/docs-5.6.0/release.html), ROCm 5.7 is the
final release for gfx906 GPUs in a fully supported state.

* ROCm 6.0 shows MI50s as "under maintenance" for
  [Linux](../about/compatibility/linux-support.md) and
  [Windows](../about/compatibility/windows-support.md)

* No new features and performance optimizations will be supported for the gfx906 GPUs beyond
  ROCm 5.7.

* Bug fixes and critical security patches will continue to be supported for the gfx906 GPUs until Q2
  2024 (end of maintenance \[EOM] will be aligned with the closest ROCm release).

* Bug fixes during the maintenance will be made to the next ROCm point release.

> * Bug fixes will not be backported to older ROCm releases for gfx906.

Reviewer: Is this for AMD contribution only or non-AMD contribution too, considering this project is open-source?

#### OS and GPU support changes

> * AMD Instinct™ MI300A and MI300X Accelerator support is enabled for supported OSes excluding RHEL9

Author: ```suggestion
* AMD Instinct™ MI300A and MI300X Accelerator support is enabled for supported OSes excluding RHEL9.
```

> * AMD Instinct™ MI300A and MI300X Accelerator support is enabled for supported OSes excluding RHEL9.

Reviewer 2: expand OSes to operating systems. 

Recommended
AMD Instinct™ MI300A and MI300X Accelerator support is enabled for supported operating systems, excluding RHEL9.

Author: Please use the suggestion feature to directly propose edits in the future, its the +- icon

* Added support for the following operating systems:
  * RHEL 9.3
  * RHEL 8.9

#### New ROCm meta package

> A new ROCm meta packages has been added for easy installation of all ROCm core packages, tools, and libraries.

Reviewer 3: ```suggestion
A new ROCm meta package has been added for easy installation of all ROCm core packages, tools, and libraries.
```

For example, the following command will install the full ROCm package: `apt-get install rocm` (Ubuntu), or `yum install rocm` (RHEL).

#### Filesystem Hierarchy Standard

ROCm 6.0 fully adopts the Filesystem Hierarchy Standard (FHS) reorganization goals.
The backward compatibility support for old file locations is removed.

#### Compiler location change

- The installation path of LLVM changes from `/opt/rocm-<rel>/llvm` to `/opt/rocm-<rel>/lib/llvm`.

>   For backbward compatibility, a symbolic link is provided to tyhe old location and will be rmoved in ROCm 7.0 or later.

Author: ```suggestion
  For backward compatibility, a symbolic link is provided to the old location and will be removed in ROCm 7.0 or later.
```

>   For backward compatibility, a symbolic link is provided to the old location and will be removed in ROCm 7.0 or later.

Reviewer 2: Recommended

For backward compatibility, a symbolic link is provided to the old location and will be removed in a future release.

Reviewer 4: @Rmalavally my last change removed saad's request for you to, instead of making comments, make suggestions. We can directly commit suggestions, which is much faster/more efficient. (see Sam's suggestion just below)

Reviewer 2: Noted.

Reviewer 2: @LisaDelaney since you are making the editorial changes, I didn't want to override your edits. Hence, recommending the changes.

- The installation path of device library bitcode has changed from `/opt/rocm-<rel>/amdgcn` to `/opt/rocm-<rel>/lib/llvm/lib/clang/<ver>/lib/amdgcn`.

>   For backward compatibility, a symbolic link is provided and will be removed in ROCm 7.0 or later.

Reviewer 3: ```suggestion
  For backward compatibility, a symbolic link is provided and will be removed in a future release.
```

#### End of support of specific distros

The following distros are planned for end of support in ROCm 6.2:

- Ubuntu 20.04.5
- SLES 15 SP4
- RHEL 7.9
- CentOS 7.9

#### Documentation

CMake support is added for the documentation in the
[ROCm repository](https://github.com/RadeonOpenCompute/ROCm).

> #### AMD Instinct™ MI50 end-of-support notice

Reviewer 2: I think we should move this to the top of the document.

> AMD Instinct MI50, Radeon Pro VII, and Radeon VII products (collectively gfx906 GPUs) enters

Reviewer 3: ```suggestion
AMD Instinct MI50, Radeon Pro VII, and Radeon VII products (collectively gfx906 GPUs) enter
```
